EpikCord/utils/types.py

```diff
@@ -0,0 +1,4 @@
+from typing import Any, Callable
+from typing import Coroutine as _Coroutine
+
+Coroutine = Callable[..., _Coroutine[Any, Any, Any]]
```

Comment:
This is actually not a coroutine, this is an async function. Async functions return coroutines.

Reply:
Ah right yeah, no wonder the Coroutine name was taken and I was reinventing it 🤦